outdated/modules/httpd.reb

```diff
@@ -112,29 +127,26 @@ sys/make-scheme [
                 spec/port-id: spec/ref/3
                 spec/actions: spec/ref/4
             ]
-            /else [
-                do make error! "Server lacking core features."
-            ]
+            default [fail "Server lacking core features."]
         ]
 
         server/locals: make object! [
-            handler: subport: open?: _
+            handler: _
+            subport: _
+            open?: _
             clients: make block! 1024
         ]
 
-        server/locals/handler: procedure [
```

Comment:
Non-stylistic change: [PROC and PROCEDURE no longer exist](https://forum.rebol.info/t/method-and-the-argument-against-procedure/710)